stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Channels.kt

```diff
@@ -0,0 +1,570 @@
+         */
+        fun channelMute() {
+            client.muteChannel("channel-type", "channel-id").enqueue {
+                if (it.isSuccess) {
+                    // Channel is muted
+                } else {
+                    Log.e(TAG, String.format("There was an error %s", it.error(), it.error().cause))
+                }
+            }
+
+            // Get list of muted channels when user is connected
+            client.setUser(
+                User("user-id"),
+                "token",
+                object : InitConnectionListener() {
+                    override fun onSuccess(data: ConnectionData) {
+                        val user = data.user
+                        // Mutes contains the list of channel mutes
+                        val mutes: List<ChannelMute> = user.channelMutes
+                    }
+                }
+            )
+
+            // Get updates about muted channels
+            client.subscribeFor<ChannelsMuteEvent> {
+                val mutes: List<ChannelMute> = it.channelsMute
+            }
+        }
+
+        /**
+         * @see <a href="https://getstream.io/chat/docs/muting_channels/?language=kotlin#query-muted-channels">Query Muted Channels</a>
+         */
+        fun queryMutedChannels() {
+            // Retrieve channels excluding muted ones
+            val offset = 0
+            val limit = 10
+            val notMutedFilter = Filters.eq("muted", false)
+            client.queryChannels(QueryChannelsRequest(notMutedFilter, offset, limit)).enqueue {
+                if (it.isSuccess) {
+                    val channels = it.data()
+                } else {
+                    Log.e(TAG, String.format("There was an error %s", it.error(), it.error().cause))
+                }
+            }
+
+            // Retrieve muted channels
+            val mutedFilter = Filters.eq("muted", true)
+            client.queryChannels(QueryChannelsRequest(mutedFilter, offset, limit)).enqueue {
+                if (it.isSuccess) {
+                    val channels = it.data()
+                } else {
+                    Log.e(TAG, String.format("There was an error %s", it.error(), it.error().cause))
+                }
+            }
+        }
+
+        /**
+         * @see <a href="https://getstream.io/chat/docs/muting_channels/?language=kotlin#remove-a-channel-mute">Remove a Channel Mute</a>
+         */
+        fun removeAChannelMute() {
+            // Unmute channel for current user
+            channelController.unmute().enqueue {
+                if (it.isSuccess) {
+                    // Channel is unmuted
+                } else {
+                    Log.e(TAG, String.format("There was an error %s", it.error(), it.error().cause))
+                }
+            }
+        }
+    }
+
+    /**
+     * @see <a href="https://getstream.io/chat/docs/query_members/?language=kotlin">Query Members</a>
+     */
+    inner class QueryMembers {
```

Comment:
If everything other than the filters are the same in the snippets here, I think it would be good to skip the copy-pasted `queryMembers` and `enqueue` parts, and just create each filter, e.g.

```kotlin
// Query members by user.name
val filterByName = Filters.eq("name", "tommaso")

// Query member by id
val filterById = Filters.eq("id", "tommaso")
...
```

Maybe we can add the full call just once at the beginning or end, to show off what that looks like, but other than that it looks like only the filters matter.